tools/check_format_compatible.sh

```diff
@@ -76,7 +76,7 @@ https_proxy="fwdproxy:8080" git fetch github_origin
 for checkout_obj in "${checkout_objs[@]}"
 do
    echo == Generating DB from "$checkout_obj" ...
-   git checkout origin/$checkout_obj
```

Comment:
My bad, too. I saw the 'origin' in the earlier PR, and spent a few seconds thinking if it should instead be parameterized instead of hardcoding to origin. But that thought never made it from my head to be a real comment on the PR :) .